taskiq/abc/broker.py

```diff
@@ -121,13 +123,18 @@ async def kiq(
         :param args: function's arguments.
         :param kwargs: function's key word arguments.
 
+        :raises SendTaskError: if we can't send task to the broker.
+
         :returns: taskiq task.
         """
         logger.debug(
             f"Kicking {self.task_name} with args={args} and kwargs={kwargs}.",
         )
         message = self._prepare_message(*args, **kwargs)
-        await self.broker.kick(message)
+        try:
+            await self.broker.kick(message)
+        except AMQPConnectionError as exc:
```

Comment:
Please catch `Exception` not specific exceptions. 

What if someone will implement kafka broker? We don't want to have kafka in dependencies in taskiq and moreover we don't want to add extra except clause.

Reply:
Changed specific exceptions to `Exception`